src/SumoCoders/FrameworkCoreBundle/Twig/TwitterBootstrap3Template.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Pagerfanta package.
+ *
+ * (c) Pablo Díez <pablodip@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SumoCoders\FrameworkCoreBundle\Twig;
+
+use Pagerfanta\View\Template\TwitterBootstrapTemplate;
+
+/**
+ * TwitterBootstrap3Template
+ */
+class TwitterBootstrap3Template extends TwitterBootstrapTemplate
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->setOptions(array('active_suffix' => '<span class="sr-only">current page</span>'));
```

Comment:
Here the array can be defined in short notation.

Reply:
fixed 82caa0e